### ISSUE TITLE
Cannot watch TypeError

### ISSUE THREAD

**Issue by pisanovo** [screepers/screeps-multimeter#25]
After following the set-up process and adding `watch-client.js` to the code base, I'm unable to use `/watch`. I call `watcher()` at the end of my game loop.

Below you can find the console output from executing `/watch status _.keys(Game.creeps).length` with some additional information:

```
*** Connecting to (https://screeps.com:443/) ...
*** Now showing Screeps console. Type /help for help.
*** Cannot watch: TypeError: Cannot read property 'expressions' of undefined
        at multimeter.api.memory.get.then.val (C:\Users\XATEV\AppData\Roaming\npm\node_modules\screeps-multimeter\plugins\watch.js:40:72)
        at processTicksAndRejections (internal/process/next_tick.js:81:5)
<<< JSON.stringify(Memory.watch)
>>> {"expressions":{},"values":{}}
<<< typeof Memory.watch
>>> object
<<< typeof Memory.watch.expressions
>>> object
*** This is Multimeter 1.8.3, which is the latest version.
<<< watcher = require("./watch-client")
>>> function() {
      if (typeof Memory.watch !== "object") {
        Memory.watch = {};
      }
      if (typeof Memory.watch.expressions !== "object") {
        Memory.watch.expressions = {};
      }
      if (typeof Memory.watch.values !== "object") {
        Memory.watch.values = {};
      }
      _.each(Memory.watch.expressions, (expr, name) => {
        if (typeof expr !== "string") return;
        let result;
        try {
          result = eval(expr);
        } catch (ex) {
          result = "Error: " + ex.message;
        }
        if (name == "console") {
          if (typeof result !== "undefined") console.log(result);      
        } else {
          Memory.watch.values[name] =
            typeof result !== "undefined" ? result.toString() : result;
        }
      });
    }
```

**Comment by pyrodogg** [comment]
It seems to be working for me on my private server (tested linux and windows), however I see the same issue if I try connecting to PTR (not currently spawned on mmo main).

**Comment by pisanovo** [comment]
Tested Linux and Windows, both seem to fail when connecting to main (shard 3).

Update: After logging the return it seems like it only contains ok but not data.

**Comment by CGamesPlay** [comment]
Resolved by #26

**Comment by SystemParadox** [comment]
Watch plugin only appears to work with shard0 at the moment - see #32.
